Firmware/mmu2_progress_converter.cpp

```diff
@@ -62,7 +62,8 @@ static const char * const progressTexts[] PROGMEM = {
 };
 
 const char * const ProgressCodeToText(uint16_t pc){
-    return ( pc <= 26 ) ? progressTexts[pc] : progressTexts[0]; // @@TODO ?? a better fallback option?
+    // @@TODO ?? a better fallback option?
+    return ( pc <= 26 ) ? static_cast<const char * const>(pgm_read_ptr(&progressTexts[pc])) : static_cast<const char * const>(pgm_read_ptr(&progressTexts[0]));
 }
```

Comment:
I saw this solution from `first_lay_cal.cpp` where we are doing a similar thing of reading a list of `PROGMEM `strings.

Reply:
so that's the way of forcing the compiler to understand, that the pointer point into PROGMEM :+1: